src/main/java/seedu/address/logic/commands/FindTagCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.PersonContainsTagPredicate;
+
+/**
+ * Lists batchmates that match all tags specified by the user.
+ * Keyword matching is case-sensitive.
+ */
+public class FindTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "findTag";
```

Comment:
The implementation looks good. 
But based on the user guide, we said to use the command word `sort team4` to find by tag. Which one should we use?

Reply:
https://ay2223s1-cs2103t-t11-4.github.io/tp/UserGuide.html#sorting-by-tags-sort

Reply:
Both are fine, but we either have to change it here or the user guide. 
Personally I think findTag is more intuitive too.

Reply:
By the way, I'm updating the user guide. So it doesn't necessarily have to be `sort`; depends on which one we prefer, `sort` or `findTag`. 😃

Reply:
I renamed cos `sort` sounds kind of weird if we are finding by tag though